test/integration/addresses.js

```diff
@@ -102,17 +102,13 @@ describe('bitcoinjs-lib (addresses)', function () {
     var address = keyPair.getAddress()
 
     dhttp({
-      method: 'POST',
```

Comment:
This line need to keep or it will result in `Error: Page not found or request method incorrect for this resource (GET)`